docs/source/special.rst

```diff
@@ -0,0 +1,21 @@
+.. role:: hidden
+    :class: hidden-section
+
+torch.special
+=============
+
+Special Operations
```

Comment:
What makes an operation "special"?

Reply:
See https://docs.scipy.org/doc/scipy/reference/tutorial/special.html for some ideas. Mentioning this is modeled after SciPy's special module would be good, too.

The formatting of "Special Operations" is a little weird since it's not a sentence or a title.

Maybe just make the first sentence of this paragraph, "The torch.special module, modeled after SciPy's special module (link), is in BETA."